### DISCUSSION TITLE
Acebase-server running on an AWS ec2 instance

### DISCUSSION THREAD

**Discussion by mmamorim** [appy-one/acebase-server#11] [General]
Hi! First of all...your project is wonderful!!! Congratulations to you for this great work!

I used it locally (localhost) and everything worked great...but I installed and ran the acebase server on an AWS ec2 machine but I'm having problems with the socket connection...it seems the http connection is working but I'm getting an error on socket connection...

Is there any specific port that I need to enable a rule for the socket connection to work?

Thank you!

**Comment by appy-one** [comment]
Do you have https enabled in your client?

**Reply by mmamorim** [reply]
I do not think so. I did a test deploying a server with express / socket.io on the same ec2 AWS machine on any port (example 4000) and the socket connection worked... Any acebase-server configuration tips that I might be doing wrong?

**Reply by appy-one** [reply]
Can you show the connection config your are using (without ip/hostname of your server), and server config?

**Reply by appy-one** [reply]
You might also want to take a look at [this issue](https://github.com/appy-one/acebase-client/issues/12), which was resolved by applying a workaround. There seems to be a bundling issue in some build environments. Let me know if that workaround works for you

**Reply by appy-one** [reply]
Do you have an update for me on this?

**Comment by appy-one** [comment]
Did you manage to get it working? If so, please share what fixed it.

**Comment by yumin-chen** [comment]
I had to wrap a Caddy HTTPS proxy on top of the Acebase-server to get it work.
